packages/objects/src/businessObjects/versioned/PermissionForStorage.ts

```diff
@@ -0,0 +1,44 @@
+import { OptInInfo } from "@objects/businessObjects/OptInInfo.js";
+import {
+  VersionedObject,
+  VersionedObjectMigrator,
+} from "@objects/businessObjects/versioned/VersionedObject.js";
+import { EPermissionType, EWalletDataType } from "@objects/enum";
+import { Permission } from "@objects/index";
+import {
+  EVMContractAddress,
+  IpfsCID,
+  TokenId,
+} from "@objects/primitives/index.js";
+import { PropertiesOf } from "@objects/utilities/index.js";
+
+export class PermissionForStorage extends VersionedObject {
+  public static CURRENT_VERSION = 1;
+  public constructor(
+    public consentAddress: EVMContractAddress,
+    public permissions: Permission[],
```

Comment:
Can Permission[] be written directly to IndexDB?

Reply:
Yes, but i rid of it and used DataPermissions in everywhere